src/language-js/postprocess.js

```diff
@@ -129,6 +172,19 @@ function postprocess(ast, options) {
           node.optional = true;
         }
         break;
+      case "TSTemplateLiteralType": {
```

Comment:
I think we should print this, not transform it.

Reply:
I have no strong opinion. Out of curiosity, why?

Reply:
Maybe we'll support `formatAST()`